src/System.Management.Automation/engine/remoting/client/RemotingErrorRecord.cs

```diff
@@ -57,32 +56,14 @@ public RemotingErrorRecord(ErrorRecord errorRecord, OriginInfo originInfo)
 
         #region ISerializable implementation
 
-        /// <summary>
-        /// Serializer method for class.
-        /// </summary>
-        /// <param name="info">Serializer information.</param>
-        /// <param name="context">Streaming context.</param>
-        public override void GetObjectData(SerializationInfo info, StreamingContext context)
-        {
-            if (info == null)
-            {
-                throw PSTraceSource.NewArgumentNullException(nameof(info));
-            }
-
-            base.GetObjectData(info, context);
-
-            info.AddValue("RemoteErrorRecord_OriginInfo", _originInfo);
-        }
-
         /// <summary>
         /// Deserializer constructor.
         /// </summary>
         /// <param name="info">Serializer information.</param>
         /// <param name="context">Streaming context.</param>
         protected RemotingErrorRecord(SerializationInfo info, StreamingContext context)
```

Comment:
Should add the obsolete attribute like in other places.

Reply:
added